src/pages/index.js

```diff
@@ -104,14 +254,9 @@ addNewCardButton.addEventListener("click", () => {
   addCardModal.open();
 });
 
-const profileEditPopup = new ModalWithForm(
-  "#profile-edit-modal",
-  handleProfileEditSubmit
-);
-
-imageModal.setEventListeners();
-profileEditPopup.setEventListeners();
-addCardModal.setEventListeners();
+avatarButton.addEventListener("click", () => {
+  handleUpdateAvatar();
```

Comment:
Please, delete it

Avatar updating is in `handleUpdateAvatarFormSubmit`

Reply:
This doesn't relate to this specific line of a code, but the avatar in general.  It also appliesto the deleteCardModal.
In your opening comments you pointed out that these elements were not aligned with the figma design.  It is visually obvious that both of these elements needed to be resized, so I made the corrections intuitively in CSS to create a better aesthetic.  However I don't see the specific dimensions anywhere in the provided figma document.  I explored this and also asked on a tutoring session and was told to format the modals based on other modals, from previous sprints.  Just want to verify that I am not missing something on figma.